spec/features/reports/identity_creates_identity_based_document_spec.rb

```diff
@@ -106,6 +116,19 @@ def when_i_open_the_protocol_dropdown
     wait_for_ajax
   end
 
+  def then_the_organization_dropdown_should_be_disabled
+    expect(page).to have_css('button.multiselect.disabled')
+  end
+
+  def when_i_open_the_organization_dropdown_and_select_an_organization
+    find('button.multiselect').click
+    check(@protocol.organization.name)
```

Comment:
I tend to avoid instance variables in my tests. Much better to define data locally within tests.